bigframes/series.py

```diff
@@ -1117,6 +1117,15 @@ def ne(self, other: object) -> Series:
         # TODO: enforce stricter alignment
         return self._apply_binary_op(other, ops.ne_op)
 
+    def items(self):
+        column_ids = self._block.value_columns
+        assert (
+            len(column_ids) == 1
+        ), f"Expected lenght of column ids to be 1, but got {len(column_ids)}."
+        bpd_series = bigframes.series.Series(self._block.select_column(column_ids[0]))
+        for index, value in zip(bpd_series.index, bpd_series.values):
```

Comment:
I would expect this to generate two separate queries, each for index and values. This is not quite ideal. Maybe take a look at dataframe.iterrows and copy that pattern.

Reply:
Per offline discussion, the query will be based on the to_pandas_batches method in the block.